### PR TITLE
PR: Added management for toolbars in new Editor windows instances.

### PR DESCRIPTION
**Pull request by dalthviz**
Fixes first point of #3790

### CODE REVIEW

**Comment by dalthviz** [comment]
A screenshot to illustrate the changes:

editorinstance

**Comment by ccordoba12** [comment]
Nice, I like it a lot!

Please merge with 3.x to fix the tests in CircleCI. This is a new service we integrated with the repo to run our tests faster and check for code style.

**Comment by ccordoba12** [comment]
@dalthviz, there are code style failures reported by `ciocheck`.

Please take a look at

https://circleci.com/gh/spyder-ide/spyder/20

to fix them :-)

File: spyder/widgets/editor.py [hunk 1]
```diff
@@ -2209,26 +2210,64 @@ def __init__(self, plugin, menu_actions, toolbar_list, menu_list,
...
+        else:
+            add_actions(menu, actions)
+    
+    def load_toolbars(self):
+        """Loads the last visible toolbars from the .ini file."""
+        
+        toolbars_names = CONF.get('main', 'last_visible_toolbars', default=[])
+        print(toolbars_names)
+        if toolbars_names:
+
```
Comment (ccordoba12) [inline spyder/widgets/editor.py]: Remove this blank line

File: spyder/widgets/editor.py [hunk 2]
```diff
@@ -2209,26 +2210,64 @@ def __init__(self, plugin, menu_actions, toolbar_list, menu_list,
...
+        toolbars_names = CONF.get('main', 'last_visible_toolbars', default=[])
+        print(toolbars_names)
+        if toolbars_names:
+            
+            dic = {}
+            for toolbar in self.toolbars:
+                dic[toolbar.objectName()] = toolbar
+                toolbar.toggleViewAction().setChecked(False)
+                toolbar.setVisible(False)
+            print(dic)
```
Comment (ccordoba12) [inline spyder/widgets/editor.py]: Remove this print

File: spyder/widgets/editor.py [hunk 3]
```diff
@@ -2209,26 +2210,64 @@ def __init__(self, plugin, menu_actions, toolbar_list, menu_list,
...
+ 
+    def get_menu(self, menu_title):
+        """Get a menu."""
+        return [menu for menu in self.menus if menu.title() == menu_title]
+
+    def get_toolbars(self):
+        """Get the toolbars."""
+        return self.toolbars
+
+    def add_actions_to_menu(self, menu_title, actions):
```
Comment (ccordoba12) [inline spyder/widgets/editor.py]: This method should be called `add_toolbars_to_menu`

File: spyder/widgets/editor.py [hunk 4]
```diff
@@ -2209,26 +2210,60 @@ def __init__(self, plugin, menu_actions, toolbar_list, menu_list,
...
                 menu = self.menuBar().addMenu(title)
                 if index == 0:
                     # File menu
                     add_actions(menu, actions+[None, quit_action])
                 else:
                     add_actions(menu, actions)
-                menus.append(menu)
+                self.menus.append(menu)
+ 
+    def get_menu(self, menu_title):
```
Comment (ccordoba12) [inline spyder/widgets/editor.py]: Please change the definition of this method to to `get_toolbar(self, toolbar_name)`

**Comment by ccordoba12** [comment]
Thanks @dalthviz, this looks good now :-)

**Pull request merged**